src/meta-srv/src/service/store/cached_kv.rs

```diff
@@ -79,6 +92,38 @@ impl LeaderCachedKvStore {
         Self::new(Arc::new(AlwaysLeader), store)
     }
 
+    /// The caller MUST ensure during the loading, there are no mutation requests reaching the `LeaderCachedKvStore`.
```

Comment:
Can the caller actually ensure this? The loading of leader cache is executed in a bg task, while the other parts in Metasrv is running at the same time, changing the etcd simultaneously.

Reply:
Currently, All mutations will be executed in Metasrv's procedures.